src/span.js

```diff
@@ -229,6 +233,13 @@ export default class Span {
         });
     }
 
+    _setTracerTags(tags: Array<Tag>): void {
+        for (let i = 0; i < tags.length; i++) {
+            let tag = tags[i];
+            this._tags.push(tag);
```

Comment:
i'm sure lodash has something like addAll, but is not more efficient than using a for loop.